src/main/java/org/ibp/api/java/dataset/DatasetService.java

```diff
@@ -353,4 +354,7 @@ void acceptDraftDataFilteredByVariable(Integer studyId, Integer datasetId,
 
 	List<MeasurementVariable> getVariablesByVariableTypes(Integer studyId, List<Integer> variableTypes);
 
+	List<PhenotypeAuditDTO> getPhenotypeAuditList(String observationUnitId, Integer variableId, Pageable pageable);
```

Comment:
I think it's better to create a new service called ObservationAuditService and move these two methods there. Rename methods to getObservationAuditList and countObservationAudit